src/UnicodePlots.jl

```diff
@@ -13,7 +15,7 @@ import MarchingCubes
 import NaNMath
 import Contour
 
-import FreeTypeAbstraction
+@lazy import FreeTypeAbstraction = "663a7486-cb36-511b-a19d-713bb74d65c9"
 import ColorTypes
 import FileIO
```

Comment:
```suggestion
@lazy import ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
@lazy import FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Reply:
Handling world age issues needs a bit of practice, but in principle, you'll very much likely hit it when you try to directly process the data if its type comes from the lazily loaded package.
Thus for `ColorTypes.RGB` and `MarchingCubes.MC`, they are unknown alien types unless we load them. -- This is also why the world age issues disappear when you try to call the same function again because the first failure actually triggers package loading, so no longer alien types.

---

I won't worry too much about MarchingCubes because it contributes very little to the loading latency using the julia-1.9 dev utility `@time_imports`.

```julia
julia> @time_imports using UnicodePlots
      0.6 ms  ┌ Compat
      0.9 ms  ┌ LazyModules
      4.6 ms    ┌ OrderedCollections
     42.4 ms  ┌ DataStructures
      0.4 ms  ┌ NaNMath
     22.4 ms  ┌ FixedPointNumbers
    302.2 ms  ┌ StaticArrays
     44.5 ms    ┌ ChainRulesCore
     45.7 ms  ┌ ChangesOfVariables
      0.7 ms    ┌ InverseFunctions
      3.4 ms    ┌ DocStringExtensions 67.89% compilation time
      3.3 ms    ┌ IrrationalConstants
      0.7 ms    ┌ DataAPI
      0.3 ms    ┌ StatsAPI
      0.3 ms    ┌ SortingAlgorithms
      0.7 ms    ┌ LogExpFunctions
      5.5 ms    ┌ Missings
     36.1 ms  ┌ StatsBase 14.35% compilation time
      1.0 ms  ┌ ConstructionBase
      0.7 ms  ┌ MarchingCubes
     30.7 ms  ┌ Crayons
    152.9 ms  ┌ Unitful
      1.3 ms  ┌ Contour
    107.9 ms  ┌ ColorTypes 4.23% compilation time
    809.8 ms  UnicodePlots 3.90% compilation time
```